packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,13 +1,8 @@
-## 0.3.0+8
+## 0.3.1
 
 * Fixes a memory leak on iOS.
-
-## 0.3.0+7
-
 * Minor fixes for new analysis options.
-
-## 0.3.0+6
-
+* Adds the `identifier` and `type` fields to the `SKProductDiscountWrapper` to reflect the changes in the [SKProductDiscount](https://developer.apple.com/documentation/storekit/skproductdiscount?language=objc) in iOS 12.2. 
```

Comment:
Looks like there is some rebase issue, need to rebase again

Reply:
> Can you guys handle this?

It's non-trivial for us to do since you used your `main` branch, which we can't directly edit.